.gitignore

```diff
@@ -25,3 +25,5 @@ server/certificate/server.csr
 server/certificate/server.key
 tools/
 client/app/app-scss.css
+.DS_Store
```

Comment:
DS_Store? What is that? O_O

Reply:
It's an invasive osx system file commonly ignored in .gitignore files ;)

Reply:
Oh.  Ignore my comment then ^^